site/docs/recipe_hello_quarks.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Recipe 1. Hello Quarks!
+---
+
+Quarks' pure Java implementation is a powerful feature which allows it to be run on the majority of JVM-compatible systems. It also has the added benefit of enabling the developer to develop applications entirely within the Eclipse and Intellij ecosystems. For the purposes of this (and future) recipes, it will be assumed that the developer is using Eclipse. To begin the Hello World recipe, create a new project and import the necessary libraries as outlined in the [getting started guide](quarks-getting-started). Next, write the following template application:
+
+``` java
+    public static void main(String[] args) {
+        DirectProvider dp = new DirectProvider();
+        Topology top = dp.newTopology();
+    }
+```
+
+The *DirectProvider* is an object which allows the user to submit and run the final application. It also creates the *Topology* object, which gives the developer the ability to define a stream of strings.
```

Comment:
Would it be better to have API references as inline code blocks? For example, `DirectProvider` instead of *DirectProvider*?